plugins/event_source/cmd.py

```diff
@@ -0,0 +1,92 @@
+"""
+cmd.py
+
+An ansible-rulebook event source plugin for running a custom command.
+It can send events to the rule engine from the stdout.
+
+Arguments:
+    command: str
+        The string with the command to execute
```

Comment:
Is it intended to be an analogue of `ansible.builtin.command` module ?

If yes, the ansible's counterpart expects a string or a list, but regardless of the input format it is processed as a list of arguments, not a shell script.

If no, the documentation is misleading, because it's not a command, but a shell script with the respective rules of escaping arguments, environment variables substitution, etc.

Reply:
Agree. Changed to process_exec to be a real command.